cmd/vicadmin/server.go

```diff
@@ -550,7 +553,10 @@ func deriveErrorMessage(err error) string {
 func (s *server) index(res http.ResponseWriter, req *http.Request) {
 	defer trace.End(trace.Begin(""))
 	ctx := context.Background()
-	sess, err := s.getSessionFromRequest(req)
+	sess, err := s.getSessionFromRequest(ctx, req)
+	if err != nil {
+		panic(err.Error())
```

Comment:
And actually -- @anchal-agrawal 's change handles the `nil` case implicitly so we don't actually even need error checking here because the rest of the function will Do the Right Thing regardless.